rholang/src/test/scala/coop/rchain/rholang/interpreter/accounting/CostAccountingSpec.scala

```diff
@@ -32,25 +36,36 @@ class CostAccountingSpec extends FlatSpec with Matchers with PropertyChecks with
     implicit val logF: Log[Task]           = new Log.NOPLog[Task]
     implicit val metricsEff: Metrics[Task] = new metrics.Metrics.MetricsNOP[Task]
     implicit val noopSpan: Span[Task]      = NoopSpan[Task]()
-    implicit val ms: Metrics.Source        = Metrics.BaseSource
+    implicit val parallel: Parallel[Task]  = new CatsParallelForTask()
 
     val resources = for {
       dir     <- Resources.mkTempDir[Task]("cost-accounting-spec-")
       costLog <- Resource.liftF(costLog[Task]())
-      cost    <- Resource.liftF(CostAccounting.emptyCost[Task](implicitly, metricsEff, costLog, ms))
-      sar     <- Resource.liftF(Runtime.setupRSpace[Task](dir, 1024L * 1024 * 1024))
+      space   <- Resource.liftF(RhoRuntime.setupRhoRSpace[Task](dir, 1024L * 1024 * 1024))
       runtime <- {
-        implicit val c = cost
-        Resource.make(Runtime.create[Task]((sar._1, sar._2), Nil))(_.close())
+        implicit val c: FunctorTell[Task, Chain[Cost]] = costLog
+        Resource.make(
+          RhoRuntime
+            .createRhoRuntime[Task](space, Seq.empty, false)(
+              implicitly,
+              logF,
+              metricsEff,
+              noopSpan,
+              parallel,
+              c
```

Comment:
Can this be simplified without writing implicit arguments? And _parallel_ should then already be in the scope.

Reply:
Another costLog https://github.com/rchain/rchain/blob/dev/rholang/src/main/scala/coop/rchain/rholang/interpreter/accounting/package.scala#L54-L58 is in the package scope `coop.rchain.rholang.interpreter.accounting`. It would cause `ambiguous implicit values` if I don't explicit specify the value.

Reply:
Cool. I didn't know that.